cvat-core/src/api.ts

```diff
@@ -374,6 +375,26 @@ function build(): CVATCore {
                 },
             },
         },
+        requests: {
+            async list() {
+                const result = await PluginRegistry.apiWrapper(cvat.requests.list);
+                return result;
+            },
+            async cancel(rqID: string) {
+                const result = await PluginRegistry.apiWrapper(cvat.requests.cancel, rqID);
+                return result;
+            },
+            async listen(
+                rqID: string,
+                options?: {
+                    callback?: (request: Request) => void,
```

Comment:
Still not clear why callback is optional

Reply:
Made it required